### DISCUSSION TITLE
Warehouse (classic) round 30 speedrun

### DISCUSSION THREAD

**Discussion by Kingjaxgaming420** [nzp-team/nzportable#959] [Records and Speedruns]
![Screenshot (7)](https://github.com/user-attachments/assets/250062a7-16a3-4c19-b22d-5ae7737ca964)
This is NOT the wr, the wr is 41:00. Only 2 seconds off tho

**Comment by Stormshadow145** [comment]
You should activate windows. in all seriousness good job

**Comment by Kingjaxgaming420** [comment]
![Screenshot (9)](https://github.com/user-attachments/assets/cc6b902e-e2c6-4ca8-8aea-d0f4f5b3f4cb)
almost a sub 40, but i beat wr by almost a minute

**Comment by Taswp** [comment]
World record competition and round speed running? this is getting great. Just getting into this

**Comment by Kingjaxgaming420** [comment]
![image](https://github.com/user-attachments/assets/77ef5bda-831d-4eac-9b91-fa51bcc6facf)
First sub 40
